aquadoggo_cli/src/main.rs

```diff
@@ -48,13 +52,22 @@ struct Opt {
     /// - "456def" with log_ids 6 7
     #[structopt(short = "A", parse(try_from_str = parse_key_val), number_of_values = 1)]
     public_keys_to_replicate: Vec<(String, Vec<u64>)>,
+
+    /// Enable mDNS for peer discovery over LAN.
+    #[structopt(short, long)]
+    mdns: bool,
+
+    /// Enable ping for connected peers (send and receive ping packets).
+    #[structopt(long)]
+    ping: bool,
```

Comment:
As mentioned above I think these could be enabled by default in the client application?

Reply:
Imagining people playing w. it (without reading the docs ..) and wondering why peers can't find each other "automagically" :laughing: